### ISSUE TITLE
Rebuild new docker image for omxplayer and install missing libraries

### ISSUE THREAD

**Issue by M4rich** [maddevsio/virtual-okno#3]
If run command for omxplayer 
/usr/bin/omxplayer.bin -o hdmi tcp://192.168.0.12:5001
get error 

> /usr/bin/omxplayer.bin: error while loading shared libraries: libswresample.so.1: cannot open shared object file: No such file or directory

Dockerfile 
https://github.com/maddevsio/virtual-okno/blob/dev/Docker/Dockerfile-omxpalyer

docker-compose.yml
https://github.com/maddevsio/virtual-okno/blob/dev/Docker/docker-compose.yml

**Comment by khalezin** [comment]
I was unable to install the libswresample.so.1 library on debian: jessie. Only libswresample.so.2 is available, but it is not suitable for omxplayer_0.3.6
I tried omxplayer_0.3.7, but there is another problem with it, it needs the libavutil.so.55 library, and only libavutil.so.54 is available

I found two solutions to run omxplayer -o hdmi tcp: // $ {RECIVE_IP}: 5009 in the container.
1. Forward volume with libraries from the host to the container:
 docker run -it --rm --device / dev / vchiq: / dev / vchiq -v / opt / vc / lib /: / opt / vc / lib --name khalezin dymokd / omxplayer: buster bash
Then the stream starts correctly.

2. If you do not forward libraries, then there is a problem with one of them libbcm_host.so
It can be copied to the container from the host in dockerfile:
COPY /opt/vc/lib/libbcm_host.so / opt / vc / lib /
Then the stream also starts correctly.
